rsync/config.yaml

```diff
@@ -26,15 +26,17 @@ schema:
   private_key_file: match(^/ssl/.+)
   username: str
   folders:
-    - source: str
-      destination: str
+    - local: str
+      remote: str
       options: str?
+      direction: str?  # 'push' (local to remote) ou 'pull' (remote to local)
```

Comment:
Could you try to use `match(...)` with a regex for "pull" or "push" here?
That way it can be ensured to get only those two inputs.